src/components/Dashboard/TodoList.vue

```diff
@@ -81,8 +83,13 @@
                 />
               </div>
 
-              <div class="list-item__subtitle">
-                <div v-if="task.subtitle && !task.bcolErrObj" class="todo-subtitle">
+              <div v-else class="list-item__subtitle">
```

Comment:
We have to be careful in this large if/else-if code block that the cases are mutually exclusive. (Bit by bit as I work on the Todo list and History list, I am refactoring to ensure cases are mutually exclusive by cleaning up the logic to make it so.)

I think we haven't yet seen a conflict between the previous case (line 82) and this subtitle block but I think this v-else needs to be here. The unit tests pass, and as long as they verify the difference scenarios then we should be OK. Thoughts?

Reply:
I think if they are meant to be mutually exclusive then this would be the best approach!

Reply:
I think if you're displaying the BCOMP AR "verify" checkbox then you don't want to show any other subtitle or buttons, right?

Reply:
We have a subtile for the BCOMP AR above the checkbox and we also want to show the button but in it's disabled state until the user "confirms" there information is up to date.

![Screen Shot 2020-07-07 at 9 24 14 AM](https://user-images.githubusercontent.com/53542131/86812237-a6a64100-c033-11ea-9713-f56c7f4a587e.png)